<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>681</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>FlightsðŸ›«</string>
    </property>
    <widget class="QRadioButton" name="one_way_radio">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Oneway</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="from_one_way_combo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="currentText">
      <string>From</string>
     </property>
     <item>
      <property name="text">
       <string>From</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="To_one_way_combo">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>60</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>To</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>60</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>â†”</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="date_one_way">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>60</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="class_one_way">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="currentText">
      <string/>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="search_button_one_way">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>100</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>ðŸ”ŽSearch</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>35</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Class:</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="date_one_way_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>60</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>681</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>FlightsðŸ›«</string>
    </property>
    <widget class="QRadioButton" name="round_trip_radio">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Roundtrip</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>60</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>â†”</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="arrivale_date_round">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>60</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="class_round_trip">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="currentText">
      <string/>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="from_round_trip_combo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>From</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="search_round_button">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>100</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>ðŸ”ŽSearch</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="come_back_round_date">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>60</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="To_round_trip_combo">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>60</y>
       <width>191</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>To</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>35</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Class:</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>723</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
